src/data/security-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const securityQuiz = [
+  {
+    Question:
+      "Which way is the most common way to leak sensitive information from your device?",
```

Comment:
Let's change this question to say what a man in the Middle attack is

```suggestion
      "In digital security, what is a Man in the Middle Attack?",
```